tests/utils/setup_test.go

```diff
@@ -176,7 +176,7 @@ func TestSetupIngress(t *testing.T) {
 	_, err = ExecuteCommand("helm repo update ingress-nginx")
 	require.NoErrorf(t, err, "cannot update ingress-nginx helm repo - %s", err)
 
-	_, err = ExecuteCommand(fmt.Sprintf("helm upgrade --install %s ingress-nginx/ingress-nginx --set fullnameOverride=%s --set controller.service.type=ClusterIP --namespace %s --wait",
+	_, err = ExecuteCommand(fmt.Sprintf("helm upgrade --install %s ingress-nginx/ingress-nginx --set fullnameOverride=%s --set controller.service.type=ClusterIP --set controller.progressDeadlineSeconds=0 --namespace %s --wait",
```

Comment:
I think this would result in the exact same rendering, shouldn't this be a different value or `null`?
```suggestion
	_, err = ExecuteCommand(fmt.Sprintf("helm upgrade --install %s ingress-nginx/ingress-nginx --set fullnameOverride=%s --set controller.service.type=ClusterIP --set controller.progressDeadlineSeconds=null --namespace %s --wait",
```